cluster/cluster.py

```diff
@@ -0,0 +1,64 @@
+"""
+Cluster job
+"""
+
+from psycopg2 import connect, sql
+
+from common.config import Config
+from common.logging import get_logger, init_logging
+
+CFG = Config()
+
+LOGGER = get_logger(__name__)
+
+
+def get_conn():
+    """Returns DB connection"""
+    return connect(
+        dbname=CFG.db_name,
+        user=CFG.db_user,
+        password=CFG.db_pass,
+        host=CFG.db_host,
+        port=CFG.db_port,
+        sslmode=CFG.db_ssl_mode,
+        sslrootcert=CFG.db_ssl_root_cert_path,
+    )
+
+
+def cluster() -> None:
+    """Cluster `system_platform` and all `system_vulnerabilities_active_*` tables"""
+    LOGGER.info("Starting cluster job")
+
+    conn = get_conn()
+    cur = conn.cursor()
+
+    LOGGER.info("Clustering system_platform table")
+    cur.execute("""CLUSTER system_platform USING system_platform_rh_account_id_idx""")
+
+    if CFG.cluster_system_vulnerabilities:
+        cur.execute(
+            """SELECT child.relname AS child
+                    FROM pg_inherits
+                    JOIN pg_class parent ON pg_inherits.inhparent = parent.oid
+                    JOIN pg_class child  ON pg_inherits.inhrelid  = child.oid
+                    WHERE  parent.relname = 'system_vulnerabilities_active'"""
+        )
+
+        sys_vulns_active_tables = list(cur.fetchall())
+        for (table,) in sys_vulns_active_tables:
+            LOGGER.info("Clustering %s table", table)
+            query = sql.SQL("CLUSTER {} USING {}").format(
+                sql.Identifier(table),
+                sql.Identifier(table + "_rh_account_id_cve_id_idx1"),
```

Comment:
I am afraid that the indexes can change name, we could get that index name dynamically by using.
```
select indexname, indexdef from pg_indexes where tablename = 'system_vulnerabilities_active_81' and indexdef LIKE '%(rh_account_id, cve_id)%';
```